director/accounts/models.py

```diff
@@ -38,45 +69,62 @@ def __str__(self) -> str:
         return self.name if self.name else 'Account #{}'.format(self.pk)
 
 class Team(models.Model):
+    """
+    Team is can be formed by one or more Users. Each User can be a member of multiple Teams.
+    Each Team is linked to exactly one Account.
+    """
     account = models.ForeignKey(
         Account,
-        on_delete=models.CASCADE
+        on_delete=models.CASCADE,
+        help_text = 'Account to which the Team is linked to. Each Team can be linked to only one account.'
     )
 
     name = models.TextField(
         blank=False,
-        null=False
+        null=False,
+        help_text = 'The name of the team (required).'
     )
 
     description = models.TextField(
-        blank=True, null=True
+        blank=True,
+        null=True,
+        help_text= 'Team description (optional).'
     )
 
     members = models.ManyToManyField(
-        'auth.User'
+        'auth.User',
+        help_text = 'Team members. Each User can be a member of multiple Teams.'
     )
 
     def __str__(self) -> str:
         return self.name
 
 
 class AccountPermission(models.Model):
+    """
+    Model implementing types of the permissions to the `Account`.
+    """
     type = models.TextField(
         null=False,
         blank=False,
         choices=AccountPermissionType.as_choices(),
-        unique=True
+        unique=True,
+        help_text= 'Permissions to the Account: Administer or Modify (required).'
     )
 
     def __str__(self) -> str:
         return self.type
 
 
 class AccountRole(models.Model):
-    name = models.TextField()
```

Comment:
I have removed `name = models.TextField()` as I *think*  this field is not needed (?). @beneboy  please review and let me know if I should put it back again.

Reply:
It is necessary, to give the AccountRole some kind of name,  i.e. "Manager", "Administrator" etc

Reply:
@beneboy Thanks! I was kind of thinking that but then getting confused because it was empty. Can add the code to it analogous to how `type` field is defined in `AccountPermission`?

Reply:
I thought about:

```
class AccountRoleType(EnumChoice):
     ADMINISTRATOR = "administrator"
     MEMBER = "member"

type = models.TextField(
        null=False,
        choices=AccountRoleType.as_choices(),
        unique=True,
        help_text= 'Roles to the Account: Administrator or Member (required).'
    )
```

Reply:
I don't think that is necessary. The reason for using Enums for the types is that we use id of the type (`modify` or `administer`) to load/assign permissions so we want them to be standardised.

It's redundant to add a restriction to the `Role` models because we're not identifying permissions by role, just using role to group together related permissions under a sensible name.

Reply:
OK. Thanks for the explanation, @beneboy